frontend/src/components/VisaStatusTabs.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Tabs, Row, Col, Button } from "antd";
+import { useTranslation } from "react-i18next";
+import { changeTabAndSetCookie } from "../redux/visastatusTabSlice";
+import VisaStatusOverviewList from "./VisaStatusOverviewList";
+import EmbassySelector from "./EmbassySelector";
+import "./VisaStatusTabs.less";
+import { OverviewChartByMinute, OverviewChartByDate } from "./VisaStatusOverviewChart";
+import { setCookie, getCookie } from "../utils/cookie";
+
+const { TabPane } = Tabs;
+
+export default function VisaStatusTabs() {
+    const [t] = useTranslation();
+    const dispatch = useDispatch();
+    const chosenKey = useSelector(state => state.visastatusTab);
+    const visaTypeDetails = useSelector(state => state.metadata.visaTypeDetails);
+    const [showChart, setShowChart] = useState(getCookie("showChart", false));
+    useEffect(() => {
+        setCookie("showChart", showChart);
+    }, [showChart]);
+    const SelectOverviewType = () => {
+        return (
+            <Button type="primary" onClick={() => setShowChart(!showChart)}>
+                {t(showChart ? "filterOverviewClose" : "filterOverviewOpen")}
+            </Button>
+        );
+    };
+
+    return (
+        <>
+            <Tabs
+                activeKey={chosenKey}
+                onChange={activeKey => dispatch(changeTabAndSetCookie(activeKey))}
+                type="card"
+                size="large"
+                renderTabBar={(props, DefaultTabBar) => <DefaultTabBar {...props} className="autofill-tab-bar" />}
+            >
+                {Array.from("FBHOL").map(visaType => (
+                    <TabPane tab={visaTypeDetails[visaType]} key={visaType} />
+                ))}
+            </Tabs>
```

Comment:
I think you have made a good point here we probably don't need the tab. We can get rid of it and create a responsive button group widget with the same _selected tab_ control. And this will also help solving the problem of rendering the TabBar in mobile screen.